marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mergeWarnings.cy.tsx

```diff
@@ -144,12 +144,6 @@ describe("Validate Merge warnings", () => {
     mergeRuleModal.saveButton();
     mergeRuleModal.alertContent().contains("Warning: The current merge settings might produce merged documents that are inconsistent with the entity type In the entity type Person, the property or properties Address allows only a single value. In every merge rule for the property Address set Max Values or Max Sources to 1.");
     mergeRuleModal.alertContent().contains("Please set max values for property to 1 on merge to avoid an invalid entity instance.");
-    //Will uncomment once DHFPROD-7452 is fixed
```

Comment:
7452 was done. Would this mean that we can uncomment this part?

Reply:
https://project.marklogic.com/jira/browse/DHFPROD-10190

I create a task to review.